stream_alert/alert_processor/outputs.py

```diff
@@ -353,3 +353,61 @@ def dispatch(self, **kwargs):
         )
 
         self._log_status(resp)
+
+@output
+class LambdaOutput(AWSOutput):
+    """LambdaOutput handles all alert dispatching for AWS Lambda"""
+    __service__ = 'aws-lambda'
+
+    def get_user_defined_properties(self):
+        """Get properties that must be asssigned by the user when configuring a new S3
+        output.  This should be sensitive or unique information for this use-case that needs
+        to come from the user.
+
+        Every output should return a dict that contains a 'descriptor' with a description of the
+        integration being configured.
+
+        S3 also requires a user provided bucket name to be used for this service output. This
+        value should not be masked during input and is not a credential requirement
+        that needs encrypted.
+
+        Returns:
+            [OrderedDict] Contains various OutputProperty items
+        """
+        return OrderedDict([
+            ('descriptor',
+             OutputProperty(description='a short and unique descriptor for this Lambda function '
+                                        'configuration (ie: abbreviated name)')),
+            ('aws_value',
+             OutputProperty(description='the AWS Lambda function name with the optional qualifier '
+                                        'to use for this Lambda configuration '
+                                        '(ie: function_name:qualifier)',
+                            input_restrictions={' '})),
+        ])
+
+    def dispatch(self, **kwargs):
+        """Send alert to a Lambda function
```

Comment:
This says you're sending an alert to Lambda, yet the paragraph below says its writing data to a file in a folder structure?